icons/filezilla/filezilla-plain-wordmark.svg

```diff
@@ -1 +1 @@
2-.078-.059-.21-.239-.289-.403a1.591 1.591 0 0 0-.344-.457c-.11-.09-.195-.207-.195-.261 0-.055-.102-.168-.227-.25a17.007 17.007 0 0 1-1.02-.747c-.116-.093-.433-.21-.698-.253a3.291 3.291 0 0 1-.813-.262c-.48-.262-2.488-.266-2.973-.008m55.118-.117c-.067.023-.27.113-.454.2-.18.085-.605.179-.949.21-.476.04-.648.09-.754.223-.074.097-.308.207-.511.246-.664.129-1.004.683-.63 1.02.11.12.184.269.216.429.078.434.507.484.851.102.086-.102.34-.2.625-.25.262-.043.55-.149.637-.23.14-.134.36-.161 1.555-.185l1.386-.027.52.266c.285.144.555.265.601.265.13 0 .868.754.868.887 0 .066.09.219.199.344.23.262.316.715.328 1.742.012.984.078.945-1.7 1.016-1.284.05-1.538.086-1.765.23-.176.113-.508.195-1.016.242-.593.059-.793.114-.941.258-.102.098-.246.184-.32.184-.278.039-.54.164-.746.36a.492.492 0 0 1-.297.155c-.192 0-1.621 1.434-1.7 1.707-.058.172-.14.336-.242.485-.367.566-.351 3.777.02 4.07.101.078.223.29.27.461.046.176.148.34.234.367.082.028.191.145.242.262.05.117.148.215.21.215.067 0 .235.097.372.222.14.121.395.258.566.309.184.055.352.148.5.273.293.278 3.168.286 3.536.004.128-.097.414-.226.64-.285.227-.058.457-.18.52-.273.062-.09.164-.164.226-.164.117 0 .223-.094 1.004-.868.61-.609.715-.53.828.606.078.785.137.828 1.23.879.618.027.763.012.825-.098.145-.273.004-11.195-.148-11.484-.075-.145-.184-.512-.239-.817-.058-.308-.168-.636-.246-.73a1.852 1.852 0 0 1-.254-.504c-.062-.184-.18-.367-.258-.41-.082-.047-.148-.125-.148-.18 0-.152-.648-.734-.934-.84a.978.978 0 0 1-.382-.257c-.086-.114-.305-.196-.704-.262a6.015 6.015 0 0 1-1.011-.262c-.418-.16-2.278-.238-2.656-.113m-74.372.45-.207.062-.02 7.617-.019 7.62.227.063c.266.07 2.105.196 2.164.149.023-.02.043-3.524.043-7.79v-7.753l-.988-.016c-.547-.004-1.082.016-1.196.047m48.578.027c-.105.11-.152 14.957-.05 15.243.043.105.183.136.683.136.344.004.79.04.98.078.298.059.368.047.43-.082.086-.168.125-15.25.043-15.39-.074-.13-1.96-.114-2.086.015m-26.64 1.41c.16.094.324.184.492.266.328.16 1.273 1.16 1.453 1.54.09.187.188.374.285.558.094.195.149.41.16.625 0 .18.075.547.165.816.367 1.09.558 1.043-4.211 1.059-4.676.015-4.34.086-4.149-.867.05-.235.086-.477.113-.715 0-.09.094-.278.207-.418.114-.14.208-.348.208-.461 0-.207.21-.512.718-1.027.149-.149.27-.313.27-.368 0-.05.074-.12.168-.152a.64.64 0 0 0 .289-.25c.066-.105.277-.242.472-.3.192-.063.415-.184.497-.274.195-.215 2.515-.242 2.859-.035m56.957 6.12.227.177.02 1.347c.019 1.172.003 1.39-.134 1.711-.097.234-.172.48-.219.734-.039.25-.12.41-.253.504-.11.075-.2.204-.2.286 0 .203-1.058 1.293-1.254 1.293-.054 0-.242.105-.421.234-.32.23-.348.234-1.586.234l-1.262.004-.234-.238c-.133-.129-.282-.234-.332-.234-.122 0-.68-.614-.68-.75 0-.055-.094-.207-.207-.336-.258-.29-.32-2.32-.082-2.532.07-.058.195-.273.281-.472.086-.203.23-.41.324-.465a.509.509 0 0 0 .215-.254c.027-.09.168-.215.317-.277a1.04 1.04 0 0 0 .367-.254c.054-.082.265-.172.465-.203.199-.036.449-.137.55-.227.235-.21.254-.215 1.727-.371 1.535-.16 2.078-.14 2.371.09" style="stroke:none;fill-rule:evenodd;fill:#a7171b;fill-opacity:1"/></g></g></svg>
```

Comment:
There's some clip-paths and groups and stuff here that's not needed.
I tried removing it and optimizing using SVGO... but it turns out weird with seemingly a copyright icon to the right.
![image](https://github.com/devicons/devicon/assets/43886029/3983f150-a953-4bd3-a7ee-f8406b62a746)


I think this version needs to be re-done based on the original-wordmark version